subsys/bluetooth/host/iso.c

```diff
@@ -956,21 +902,25 @@ static struct net_buf *hci_le_set_cig_params(struct bt_iso_create_param *param)
 		}
 
 		if (!qos->tx) {
-			/* Use RX PHY if TX is not set (disabled) */
-			cis->m_phy = qos->rx->phy;
+			/* Use RX PHY if TX is not set (disabled)
+			 * to avoid setting invalid values
+			 */
+			cis_param->m_phy = qos->rx->phy;
 		} else {
-			cis->m_sdu = sys_cpu_to_le16(qos->tx->sdu);
-			cis->m_phy = qos->tx->phy;
-			cis->m_rtn = qos->tx->rtn;
+			cis_param->m_sdu = sys_cpu_to_le16(qos->tx->sdu);
+			cis_param->m_phy = qos->tx->phy;
+			cis_param->m_rtn = qos->tx->rtn;
 		}
 
 		if (!qos->rx) {
-			/* Use TX PHY if RX is not set (disabled) */
-			cis->s_phy = qos->tx->phy;
+			/* Use TX PHY if RX is not set (disabled)
+			 * to avoid setting invalid values
+			 */
+			cis_param->s_phy = qos->tx->phy;
```

Comment:
Ditto for tx here.

Reply:
Ditto: https://github.com/zephyrproject-rtos/zephyr/pull/37778#discussion_r693981261